src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/sensors/Sensors.scala

```diff
@@ -163,10 +163,10 @@ class Sensors @Inject()(eventProcessor: EventProcessor, sensorRepository: Sensor
         )
       ) match {
         case Success(s) => sensors.put(sensor.id, s)
-        case Failure(f) => logger.error(s"Couldn't create Recurring sensor for sensor (#${sensor.id}).", f)
+        case Failure(f) => logger.error(s"Could not create Recurring sensor for sensor (#${sensor.id}).", f)
       }
     case _ =>
-      logger.error(s"Couldn't not fid sensor implementation (#${sensor.id}).", sensor.properties.sensorType)
+      logger.error(s"Could not not fid sensor implementation (#${sensor.id}).", sensor.properties.sensorType)
```

Comment:
No.. It should be `Could not find`, but it is `Could not not fid`

Reply:
Fixed